pom.xml

```
@@ -0,0 +1,350 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>org.example</groupId>
+	<artifactId>example-osgi-portlet</artifactId>
+	<version>1.0.0-SNAPSHOT</version>
+	<name>Example OSGi Portlet</name>
+	<url>https://github.com/qbicsoftware/example-osgi-portlet</url>
+
+	<packaging>${packaging.type}</packaging>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<maven.compiler.source>11</maven.compiler.source>
+		<maven.compiler.target>11</maven.compiler.target>
+
+		<liferay.version>7.3.5</liferay.version>
+		<vaadin.version>8.11.1</vaadin.version>
+		<vaadin.plugin.version>8.11.1</vaadin.plugin.version>
```

Latest version of Vaadin 8 is 8.12.0 which also works with this example portlet.

```suggestion
		<vaadin.version>8.12.0</vaadin.version>
		<vaadin.plugin.version>8.12.0</vaadin.plugin.version>
```